docs/package-management.md

```diff
@@ -4,8 +4,9 @@ Melange can consume packages from both the [npm
 registry](https://www.npmjs.com/) and the [opam
 repository](https://opam.ocaml.org/packages/).
 
-- For Melange libraries and bindings (compile-time dependencies), use
-  [opam](https://opam.ocaml.org/).
+- For Melange libraries and bindings (compile-time dependencies), use one of
```

Comment:
Thanks. I realize now that after this change, the rest of the content below is a bit strange, because we only refer to opam, e.g.

https://github.com/melange-re/documentation-site/blob/8125480125dc7a290302a22abdf0b296cc90480e/docs/package-management.md?plain=1#L14

or

https://github.com/melange-re/documentation-site/blob/8125480125dc7a290302a22abdf0b296cc90480e/docs/package-management.md?plain=1#L23-L24

or 

https://github.com/melange-re/documentation-site/blob/8125480125dc7a290302a22abdf0b296cc90480e/docs/package-management.md?plain=1#L27